lib/ui/index.js

```diff
@@ -79,6 +79,10 @@ UI.prototype.writeLine = function(data, writeLevel) {
   if (this.writeLevelVisible(writeLevel)) {
     this.write(data + EOL);
   }
+
+  if (/ERROR/i.match(writeLevel)) {
```

Comment:
Yes. That's what we'd want, right? 

`STDOUT` to display everything (including errors), `STDERR` to display errors.